contracts/verifier/CustomSmartWalletDeployVerifier.sol

```diff
@@ -58,7 +60,17 @@ contract CustomSmartWalletDeployVerifier is IDeployVerifier, Ownable {
 
     function acceptToken(address token) external onlyOwner {
         require(token != address(0), "Token cannot be zero address");
+        require(tokens[token] == false, "Token is already accepted");
```

Comment:
doesn't `!tokens[token]` works?

Reply:
it does. it's a matter of readability and/or preference :man_shrugging:

Reply:
The require might not be needed. It adds an extra check that makes most of the calls more expensive. The only case when this require is useful is when you are trying to accept an already-accepted token. In that scenario, you MIGHT (and Im not sure) spend less gas. 
Case1: Not yet accepted, The require passes and you spent some gas in that
Case2: Already accepted, the require does not pas
Case3: the require does not exist and not yet accepted. All normal
Case4:The require does not exist and already accepted. All normal, the attribute does not change its value (from true to true)